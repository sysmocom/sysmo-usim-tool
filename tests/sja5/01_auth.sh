#!/bin/sh
. ./test-data

$TOOL -a $ADMPIN
