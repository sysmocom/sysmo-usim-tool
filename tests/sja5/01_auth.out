sysmoISIM-SJA5 parameterization tool
Copyright (c) 2023 sysmocom - s.f.m.c. GmbH

Trying to find card with ATR: 3B 9F 96 80 1F 87 80 31 E0 73 FE 21 1B 67 4A 35 75 30 35 02 59 C4
Initializing smartcard terminal...
 * Detected Card IMSI:  901700000046734
   ISIM Application installed
   USIM Application installed

Authenticating...
 * Remaining attempts: 3
 * Authenticating...
 * Authentication successful
 * Remaining attempts: 3

Done!
